### PR TITLE
if RNN's hx is None, requires_grad=False

### PR DESCRIPTION
**Pull request by donglixp**
When the initial hidden states of RNN are ``None'', we don't need to compute their gradients.

### CODE REVIEW

**Comment by soumith** [comment]
this PR is redundant, requires_grad=False is the default value, but it does make it explicit in code so I've merged it. Thanks.
